code/_core/mob/living/advanced/player/_player.dm

```diff
@@ -115,6 +115,29 @@
 
 	expiration_time = SECONDS_TO_DECISECONDS(180)
 
+/mob/living/advanced/player/get_examine_list(var/mob/living/examiner)
+	var/list/species_to_names = list(
```

Comment:
You can get the species from its id by using the define SPECIES(x).

If you want to add funny examine names, you should add that proc to the /species/ datum instead of hardcoding it in a list.

Reply:
huh, neat. Applied it